### PR TITLE
Documentation: Add minimal doc for using own CA

### PR DESCRIPTION
**Pull request by faxm0dem**
It's not clear how to advertise the CA chain as there doesn't seem to be an option to use a trusted bundle. It seems to be possible (tested with grafana 9) to concatenate the server certificate with the bundle in the same file.

### CODE REVIEW

File: docs/sources/setup-grafana/set-up-https.md [hunk 1]
```diff
@@ -222,6 +222,10 @@ To adjust permissions, perform the following steps:
    lrwxrwxrwx 1 root grafana    65 Mar 22 14:15 /etc/grafana/grafana.key -> /etc/letsencrypt/live/subdomain.mysite.com/privkey.pem
    ```
 
+### Use your own CA
+
+If you want to use your own CA or your organization's and make sure the full chain is advertized by Grafana, make sure your `cert_file` contains your signed certificate followed by the full CA bundle chain (not the opposite way around).
```
Comment (chri2547) [inline docs/sources/setup-grafana/set-up-https.md:227]:
```suggestion
If you want to use your own CA or your organization's and ensure Grafana advertises the full chain, ensure your `cert_file` contains your signed certificate followed by the full CA bundle chain.
```

**Review by chri2547 — CHANGES_REQUESTED** [review]
Made a wording suggestion. Thank you for the contribution!

**Comment by faxm0dem** [comment]
Ping for review/merge, thanks

**Comment by faxm0dem** [comment]
ping

**Review by moxious — APPROVED** [review]

**Comment by moxious** [comment]
@chri2547 mergeable from my POV - I corrected backport labels, assuming build is green, go.

**Comment by faxm0dem** [comment]
ping

**Comment by faxm0dem** [comment]
Okay folks, sorry to say this but I'm gonna just stop pushing this every month.
Apparently it's not possible for an external dev to improve your documentation. 

Cheers

**Pull request closed without merging**